en/advanced/system_startup.md

```diff
@@ -47,3 +47,17 @@ The `extras.txt` can be used to start additional applications after the main sys
 ### Starting a custom mixer
 
 By default the system loads the mixer from `/etc/mixers`. If a file with the same name exists in `/fs/microsd/etc/mixers` this file will be loaded instead. This allows to customize the mixer file without the need to recompile the Firmware.
+#### Example
+The following example shows how to add a custom aux mixer:
+  * Create a file on the SD card, `etc/mixers/gimbal.aux.mix` with your mixer
+	content.
+  * Then to use it, create an additional file `etc/config.txt` with this content:
+```
```

Comment:
@bkueng We should condition this on a parameter that you can enable to output a gimbal on the aux ports by default. The whole microSD approach is horrible from a user perspective.

I would like to automate this, move the bulk of these docs to the user guide and then only document how the gimbal driver works in the dev guide.

This is clearly unusable for an average user (and even for an average developer) as-is. It is way too complex.